server/controllers/user.controllers.js

```diff
@@ -121,9 +121,15 @@ const UserController = {
   },
 
   getAllRegisteredUsers(req, res, next) {
-    getAllRegisteredUsers()
+    const skip = req.params.count;
+    const { limit } = req.params;
```

Comment:
This also should be optional:

```suggestion
    const { limit } = req.params || 10;
```